lib/filter-mappers.js

```diff
@@ -40,7 +40,10 @@ module.exports = {
 	 * @param {any} value The value
 	 * @return {object} The value with type = search
 	 */
-	searchMapper: value => ({ type: 'search', value }),
+	searchMapper: {
```

Comment:
:bulb: 

Y si search es un caso particular de custom ? :smile_cat: 

```suggestion
	searchMapper: customTypeMapper('search'),
	customTypeMapper
```